bolt/lib/Utils/CommandLineOpts.cpp

```diff
@@ -128,6 +128,14 @@ cl::opt<bool>
                cl::desc("instrument code to generate accurate profile data"),
                cl::cat(BoltOptCategory));
 
+cl::opt<bool> Lite("lite", cl::desc("skip processing of cold functions"),
```

Comment:
More descriptive option name?

Reply:
This was the original description and looking at where it's used, I think its fairly apt - but if you have strong feelings I can make it more verbose

Reply:
Alexander, this is an existing option, just being moved around. Since we're using it from Profile component, it must be visible there. It's originally defined in Rewrite, but Profile doesn't depend on Rewrite. Hence the move to Utils which is included by both Profile and Rewrite.